Sources/SwiftMemcache/Extensions/UInt16+ResponseStatus.swift

```diff
@@ -0,0 +1,38 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the swift-memcache-gsoc open source project
+//
+// Copyright (c) 2023 Apple Inc. and the swift-memcache-gsoc project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of swift-memcache-gsoc project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+typealias ResponseStatus = UInt16
+
+extension ResponseStatus {
```

Comment:
Let's make this an enum called `ReturnCode` and nest it inside the `MemcachedResponse` type